multipatch_analysis/database/morphology.py

```diff
@@ -7,7 +7,18 @@
 
 Morphology = make_table(name='morphology', comment="Describes morphological properties of cells.", columns=[
     ('cell_id', 'cell.id', 'The ID of the cell described by each record', {'index': True, 'unique': True}),
-    ('pyramidal', 'bool', 'Whether the experimenter labeled this cell as pyramidal', {'index': True}),
+    ('cell_specimen_id', 'int', 'LIMS cell specimen ID', {'index': True}),
```

Comment:
Why's this here? If anything it should be in the cell table

Reply:
sure we can put it in the cell table.....this is how morphology is referencing everything so, a) I needed to generate it and b) I didn't want to keep having to do that